image-rhel-9.2.yaml

```
@@ -15,3 +15,9 @@ vmware-os-type: rhel8_64Guest
 # VMware hardware versions: https://kb.vmware.com/s/article/1003746
 # Supported VMware versions: https://lifecycle.vmware.com/
 vmware-hw-version: 15
+
+# see https://github.com/coreos/coreos-assembler/pull/3607
+# Defaults for AWS
+aws-imdsv2-only: false
+aws-volume-type: "gp2"
+aws-x86-boot-mode: "uefi"
```

I think this is the wrong value:

```suggestion
aws-x86-boot-mode: "bios"
```

---

I think the legacy behaviour is to not specify a boot mode parameter at all. According to https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ami-boot.html, this means that it'll be based on the instance type. I'm not sure if we set `legacy-bios` and try to boot it on a Graviton instance whether it'll break.

Probably cleaner anyway to do something similar to what we're doing for IMDSv2: https://github.com/coreos/coreos-assembler/pull/3607/files#diff-e051905e06e5eaa099fca1faa4b15ecfb54b08e54fe1763aecdaa30c77da4e34R378-R380 (i.e. leave it unset in the initial `params`, and only set the field if needed). Keeping the unset behaviour could be done here with e.g. `aws-x86-boot-mode: "default"`.

---

Oh right, Graviton are the aarch64 instance types, so I think we're fine with `legacy-bios`, yup!